CMakeLists.txt

```diff
@@ -63,6 +63,13 @@ if (NOT PYTHONLIBS_FOUND)
 endif()
 include(cmake/utils.cmake)
 
+# The building of the schemas requires some extra python packages which may or
+# may not be installed. If they aren't we provide an option to add them from
+# a python egg.
+if (BUILD_USDMAYA_SCHEMAS)
+    include(cmake/jinja.cmake)
+endif()
+
```

Comment:
This was just missed from the original PR. Makes sure that Jinja is installed in python